test/functional/lua/buffer_updates_spec.lua

```diff
@@ -1001,6 +1001,14 @@ describe('lua: nvim_buf_attach on_bytes', function()
       }
     end)
 
+    it("append with lockmarks", function ()
+      local check_events = setup_eventcheck(verify, {"AAA", "BBB"})
+
+      funcs.execute("lockmarks call append(1, 'abc')")
+
+      check_events {}
```

Comment:
where's the events?

Reply:
https://github.com/neovim/neovim/pull/14994 seems to have tests, let's go with that one :]

Reply:
Yup